steps/cordova-config/0.0.1/step.yml

```diff
@@ -0,0 +1,64 @@
+title: cordova-config
```

Comment:
The step's title is a user facing name of the step, i would recommend to use: `Manipulate cordova config.xml` for example